File: discover_ruby/github_most_popular.rb
# usage
require 'httpclient'

extheaders = {
  'User-Agent' => 'Holberton_School',
  'Authorization' => 'token 4c9a275f29ec4a763748d8d77668c90f7dea7680'
}

clnt = HTTPClient.new
puts clnt.get_content('https://api.github.com/search/repositories?q=language:ruby&sort=stars&order=desc')

File: discover_more_python/github_most_popular.py
import json

request_headers = {
  'User-Agent': 'Holberton_School',
  'Authorization': 'token 7787fe0886a53bdf16c980bb319fb09a93e2b59d'
}

import urllib2
content = urllib2.urlopen('https://api.github.com/search/repositories?q=language:python&sort=stars&order=desc').read()
print content
